client/components/dashboard/ItemTable.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState, useEffect } from 'react';
+import { DataGrid, GridRowsProp, GridColDef } from '@mui/x-data-grid';
+
+// const [tableData, setTableData] = useState([]);
+
+// useEffect(() => {
+//   fetch('/api/items/all')
+//     .then((data) => data.json())
+//     .then((data) => setTableData(data));
+// }, []);
+
+const rows = [
```

Comment:
You should be able to do a fetch request to /api/items and pull data from the json-server if you use 'npm run json-dev'